apps/address-service/README.md

```diff
@@ -1,3 +1,25 @@
+# Description
+It's a separate service for address processing.
+
+Each property is presented as the Address model instance. Each address may be found by different strings, so several AddressSource model instances may belong to each address.
+
+## Models
+- **Address**. A model containing data on the particular building's address.
+- **AddressSource**. A model containing data on the particular building's address origin.
+- **AddressInjection**. Addresses that do not exist in external providers.
+
+## Endpoints
+
+### /suggest
+Used to get a list of suggestions while the user trying to add a new property. There is an ability to use different suggestions providers. If some property still not exists in external providers, we may inject proper data before the answer is sent to the response. It may be reached by adding new AddressInjection model via admin interface.
+
+### /search
+Used to search addresses. This endpoint utilizes search plugins under the hood. The first not-empty plugin's result will send to the response.
+
+### /add
+Used to create new property during creating the same property in the condo database.
```

Comment:
? why it's here

Reply:
I trying to make the review easier and more attractive a bit. It's to make an overall understanding of how the service works :)

Reply:
The `/add` endpoint was eliminated